### PR TITLE
add/timedeltas-seconds-documentation

### LINKED ISSUE #49568: DOC: What does TimeDelta.seconds do?
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/docs/dev/reference/api/pandas.Timedelta.seconds.html#pandas-timedelta-seconds

### Documentation problem

Documentation is blank.

### Suggested fix for documentation

Write documentation for the property.

Looking at the source it seems like the definition is 

```
    @property
    def seconds(self) -> int: ...
```

Which unfortunately I do not understand. I just learned that Python has an ellipsis object. All I know is that the thing I really wanted was total_seconds(). I got a nonsense answer using seconds and wanted to figure out what happend.

### PR DESCRIPTION
**Pull request by joaopmjm**
- [x] closes #49568

### CODE REVIEW

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 1]
```diff
@@ -1035,6 +1035,37 @@ cdef class _Timedelta(timedelta):
...
+        Timedelta.components : Return all attributes with assigned values
+            (i.e. days, hours, minutes, seconds, milliseconds, microseconds,
+            nanoseconds).
+
+        Examples
+        --------
+        **Using string input**
+
+        >>> td = pd.Timedelta('1 days 2 min 3 us 42 ns')
+
```
Comment (mroeschke) [inline pandas/_libs/tslibs/timedeltas.pyx:1059]: Nit: Could you remove this blank line?

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 2]
```diff
@@ -1035,6 +1035,37 @@ cdef class _Timedelta(timedelta):
 
     @property
     def seconds(self) -> int:  # TODO(cython3): make cdef property
+        """
+        Return timedelta in number of seconds converting hours and minutes in seconds.
```
Comment (mroeschke) [inline pandas/_libs/tslibs/timedeltas.pyx:1039]: Maybe `Return the total hours, minutes, and seconds of the timedelta as seconds`

To make it clear that no other component is included.

File: pandas/_libs/tslibs/timedeltas.pyx (hunk 1 above)
Comment (joaopmjm) [reply pandas/_libs/tslibs/timedeltas.pyx]: Done

File: pandas/_libs/tslibs/timedeltas.pyx (hunk 2 above)
Comment (joaopmjm) [reply pandas/_libs/tslibs/timedeltas.pyx]: Done

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @joaopmjm

**Pull request merged**